python/alibiexplainer/alibiexplainer/explainer.py

```diff
@@ -64,8 +64,8 @@ def __init__(  # pylint:disable=too-many-arguments
         else:
             raise NotImplementedError
 
-    def load(self) -> bool:
-        pass
+    def load(self):
+        self.ready = True
```

Comment:
We can remove this as it is already defined in base https://github.com/kserve/kserve/blob/b59dde1739a29dd3d6484f6e4ea2039d85f17617/python/kserve/kserve/model.py#L173